gatsby-config.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/camelcase */
+
+module.exports = {
+  plugins: [
+    "gatsby-plugin-postcss",
+    "gatsby-plugin-react-helmet",
+    "gatsby-plugin-styled-components",
+    "gatsby-plugin-typescript",
+
+    {
+      resolve: "gatsby-plugin-google-analytics",
+      options: {
+        trackingId: "UA-117977508-2",
+        head: true,
+      },
+    },
+
+    {
+      resolve: "gatsby-plugin-manifest",
```

Comment:
Does this manifest not need/support specifying multiple icon sizes, like our previous `manifest.webmanifest`?

Reply:
It does! See the PR description and https://covid19-dashboard-lj3mj6imu.now.sh/manifest.webmanifest